### ISSUE TITLE
Request: Combobox functionality

### ISSUE THREAD

**Issue by Deebling5** [TomSchimansky/CustomTkinter#7]
Hey Tom,
Are you adding combobox functionality anytime soon? If not, how can I include Radio button and combobox within your code.

Thanks

**Comment by TomSchimansky** [comment]
Maybe I have time in the next few weeks, then I will try to do it, but I can imagine that it won't be easy because you have to open a small separate window inside the the main window on draw on this. But I will try it, the next things I will do are Checkboxes, Radiobutton and Comboxes.

But for now you can also use the standard ttk comboxes, although you cannot change the background color if you are on macOS. So you have to use them inside a frame which has the standard window background color:

```python
import tkinter
import customtkinter

customtkinter.enable_macos_darkmode()

root_tk = tkinter.Tk()  # create the Tk window like you normally do
root_tk.geometry("450x300")
root_tk.title("ComboBox Test")

frame_1 = customtkinter.CTkFrame(master=root_tk, width=380, height=250, corner_radius=15)
frame_1.place(relx=0.5, rely=0.5, anchor=tkinter.CENTER)

button_1 = customtkinter.CTkButton(master=frame_1, corner_radius=10, width=180, command=None)
button_1.place(relx=0.5, rely=0.2, anchor=tkinter.CENTER)

# ----- ComboBox ------
from tkinter import ttk

combobox_1 = ttk.Combobox(master=frame_1, values=["Option 1", "Option 2"], width=10)
combobox_1.place(relx=0.25, rely=0.6, anchor=tkinter.CENTER)
combobox_1.current(0)


frame_2 = customtkinter.CTkFrame(master=frame_1, width=160, height=100, corner_radius=15,
                                 fg_color=root_tk.cget("bg"))
frame_2.place(relx=0.7, rely=0.7, anchor=tkinter.CENTER)

combobox_2 = ttk.Combobox(master=frame_2, values=["Option 3", "Option 4"], width=10)
combobox_2.place(relx=0.5, rely=0.5, anchor=tkinter.CENTER)
combobox_2.current(0)

# ---------------------

root_tk.mainloop()
customtkinter.disable_macos_darkmode()
```


![Bildschirmfoto 2021-08-09 um 11 53 14](https://user-images.githubusercontent.com/66446067/128688925-9310259a-31ac-4ff2-93d6-f7450a6d584f.png)

**Comment by Deebling5** [comment]
Thanks Tom. It worked!! 
Will keep following you work.

**Comment by IshanJ25** [comment]
any update?

**Comment by IshanJ25** [comment]
> `◀️[ Option 1 ]▶️`

that seems innovative 👍🏻

may I suggest a `toplevel` window derived from `root` of the given dimensions, with `overedirect true` and `transparent background` to draw a `canvas` in seperate semi-dependent window with rounded corners

**Comment by TomSchimansky** [comment]
Looks great, I'm working on a ComboBox right now.
There is no inbuilt way of reseting to the default theme colors. The colors are save in `customtkinter.ThemeManagere.theme` which is a dictionary loaded from (this)[https://github.com/TomSchimansky/CustomTkinter/blob/master/customtkinter/assets/themes/blue.json] .json file. You could get the colors like this: `customtkinter.ThemeManagere.theme["color"]["button"]`, but I cannot promise that the key names will stay the same.

**Comment by TomSchimansky** [comment]
CTkComboBox and CTkOptionMenu are now available with version 4.3.0:
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkComboBox
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkOptionMenu

<img width="651" alt="Bildschirmfoto 2022-06-02 um 01 15 49" src="https://user-images.githubusercontent.com/66446067/171517978-42c048a8-7174-4d66-8771-1a852efccb7b.png">
